src/modules/RegAuth/Auth/selectors/selectors.ts

```diff
@@ -1 +1,2 @@
-export const isFetching = (state: any) => state.form.auth.isFetching;
+export const getIsFetching = (state: any) => state.form.auth.isFetching;
```

Comment:
if you want to get smthg from form reducer, you must use a redux-form selectors

https://redux-form.com/8.1.0/docs/api/selectors.md/